package.json

```
@@ -51,6 +51,6 @@
     "beachball": "^2.41.0"
   },
   "peerDependencies": {
-    "typescript": "~4.8.2"
+    "typescript": "^5.0.0"
```

```suggestion
    "typescript": "~5.0.0"
```
Should we stick to `~` where only the patch version is accepted? Since this is a JavaScript style guide that other projects depend on for consistent linting behavior, I think we might need to use `~` instead of `^` for dependencies like TypeScript and ESLint.